solutions/cses-1706.mdx

```diff
@@ -0,0 +1,73 @@
+---
+id: cses-1706
+source: CSES
+title: School Excursion
+author: Óscar Garries
+---
+## Solution
+**Time complexity:**$\mathcal{O}(N\log^2N)$
+
+The solution of the problem is based on DSU ( [Disjoin Set Union](https://usaco.guide/gold/dsu?lang=cpp) )
```

Comment:
```suggestion
The solution is based on DSU ([Disjoint Set Union](https://usaco.guide/gold/dsu?lang=cpp))
```

don't include extraneous spaces around parentheses. Also, there are many typos - you can probably detect these with Grammarly

Reply:
It's ok now ? What do you think ?